fiaas_deploy_daemon/deployer/kubernetes/ready_check.py

```diff
@@ -33,10 +33,12 @@ def __init__(self, app_spec, bookkeeper, lifecycle, lifecycle_subject, ingress_a
         self._bookkeeper = bookkeeper
         self._lifecycle = lifecycle
         self._lifecycle_subject = lifecycle_subject
+        readiness_delay = app_spec.health_checks.readiness.initial_delay_seconds
+        liveness_delay = app_spec.health_checks.liveness.initial_delay_seconds
         self._fail_after_seconds = _calculate_fail_time(
             config.ready_check_timeout_multiplier,
             app_spec.autoscaler.max_replicas,
-            app_spec.health_checks.readiness.initial_delay_seconds
+            readiness_delay if readiness_delay > liveness_delay else liveness_delay
```

Comment:
I'd prefer to use max(readiness_delay, liveness_delay) instead, clean and easy to read.